### DISCUSSION TITLE
Posthog - Open source customer data pipeline and analytics platform

### DISCUSSION THREAD

**Discussion by jhliberty** [twentyhq/twenty#8629] [Ideas]
Would be great to have twenty be a source/destination for this analytics and customer data pipeline tool.

Would love to have events from my app be able to be sent through to Twenty. Reverse could be useful as well to have all your interactions with a customer make it to the data warehouse.

**Comment by tony9823** [comment]
How can we design a seamless integration between the app and Twenty to ensure bidirectional data flow, allowing event data to enrich scheduling while customer interactions are reliably synced to the data warehouse without introducing latency or data inconsistencies?

**Comment by BOHEUS** [comment]
@jhliberty thanks for the suggestion, soon it'll be possible to write an integration once SDK is stable and v1.0 is released (in the meantime, feel free to check out docs https://docs.twenty.com/developers/extend/apps/getting-started), I'm closing this discussion in favor of collective discussion of integrations https://github.com/twentyhq/twenty/discussions/16735
